### ISSUE TITLE
add term Incus ankylosis

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10557]
**Preferred term label:**
Incus ankylosis

**Synonyms**
Incus fixation

**Definition (free text, please give PubMed ID)**
Incus ankylosis refers to congenital or acquired fixation of the incus, which is associated with conductive hearing resulting from impairment of the sound-conduction mechanism (the external auditory canal, tympanic membrane, and/or middle-ear ossicles). PMID: 24969066

**Parent term (use hpo.jax.org/app)**
Abnormality of the incus HP:0011453

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Craniometaphyseal dysplasia (omim123000), PMID16980979

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
@azankl
@lisannesw 
Adding new term: 
- ID: HP:6001071
- label: Incus ankylosis
- parents: [Abnormality of the incus (HP:0011453)](https://hpo.jax.org/app/browse/term/HP:0011453)
- pmid: PMID:24969066
- def: Incus ankylosis refers to congenital or acquired fixation of the incus, which is associated with conductive hearing resulting from impairment of the sound-conduction mechanism (the external auditory canal, tympanic membrane, and/or middle-ear ossicles).
- comment: 
- orcid: orcid:0000-0001-8612-1062
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10557
- synonyms:
